builder/builder-next/executor_unix.go

```diff
@@ -119,6 +133,9 @@ func (iface *lnInterface) Close() error {
 			if err := iface.sbx.Delete(); err != nil {
 				logrus.Errorf("failed to delete builder network sandbox: %v", err)
 			}
+			if err := os.RemoveAll(filepath.Join(iface.provider.Root, iface.sbx.ContainerID())); err != nil {
+				logrus.Errorf("failed to delete builder sandbox directory: %v", err)
```

Comment:
While at it, perhaps use `WithError()` here as well

```suggestion
				logrus.WithError(err).Error("failed to delete builder sandbox directory")
```

Reply:
I don't want this line to use a different style than the rest of the file/pkg.